src/reducers/versions.tsx

```diff
@@ -796,12 +843,19 @@ const reducer: Reducer<VersionsState, ActionType<typeof actions>> = (
     case getType(actions.collapseTree): {
       const { versionId } = action.payload;
 
+      const version = state.versionInfo[versionId];
+
+      if (!version) {
+        _log.error(`Version missing for versionId: ${versionId}`);
```

Comment:
I wonder if this should throw an actual error, rather than just logging. It shouldn't really be possible to have called `collapseTree` without having a corresponding version loaded.